tests/disaster_recovery_3.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+import os
+import shutil
+import signal
+import time
+from TestHarness import Cluster, TestHelper, Utils, WalletMgr
+from TestHarness.Node import BlockType
+
+###############################################################
+# disaster_recovery - Scenario 3
+#
+# Create integration test with 4 nodes (A, B, C, and D) which each have their own producer and finalizer. The finalizer
```

Comment:
Should be `each have their` be `each has its`?

Reply:
I rewrote the paragraph.